src/django_idom/templatetags/idom.py

```diff
@@ -16,13 +16,15 @@
 register = template.Library()
 
 
-@register.inclusion_tag("idom/view.html")
+@register.inclusion_tag("idom/component.html")
 def idom_component(_component_id_, **kwargs):
     _register_component(_component_id_)
 
+    class_ = kwargs.pop("class", "")
     json_kwargs = json.dumps(kwargs, separators=(",", ":"))
 
     return {
+        "class": class_,
```

Comment:
You could directly assign the `kwarga.pop` here and avoid the need for `class_`.

Reply:
Creating `class_` was needed in order to avoid serializing that value within the ` json.dumps`. Let me know if you disagree with this.

Reply:
I meant that you could just do:

```python
{
    "class": kwargs.pop("class", ""),
    ...
}
```

Reply:
Ah, I see what you mean now.